src/main/java/com/esri/samples/kml/play_a_kml_tour/README.md

```diff
@@ -0,0 +1,47 @@
+<h1>Play a KML Tour</h1>
+
+<p>Play tours in KML files.</p>
+
+<p><img src="PlayAKMLTour.png"/></p>
+
+<h2>Use case</h2>
+
+<p>KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia.</p>
+
+<h2 id="howtousethesample">How to use the sample</h2>
+
+<p>Click the play button to play the tour. Press again to pause the tour. Click the reset button to start again from 
+the beginning.</p>
```

Comment:
The About the Data section says this

Reply:
it does, though I think that sentence is better in the "how to use the sample" section (so that they're aware to turn up speakers etc whilst reading up about how to use the sample).